easybuild/framework/easyconfig/tools.py

```diff
@@ -321,9 +321,14 @@ def alt_easyconfig_paths(tmpdir, tweaked_ecs=False, from_prs=None):
 
     # paths where files touched in PRs will be downloaded to,
     # which are picked up via 'pr_paths' build option in fetch_files_from_pr
-    pr_paths = None
+    pr_paths = []
     if from_prs:
-        pr_paths = [os.path.join(tmpdir, 'files_pr%s' % pr) for pr in from_prs]
+        pr_paths = from_prs
+    if review_pr and review_pr not in pr_paths:
+        pr_paths.append(review_pr)
```

Comment:
Avoid side-effect of appending to `from_prs` input list, so either here do:

```python
pr_paths = pr_paths + [review_pr]
```

or above copy `from_prs` rather than doing as pass by reference:

```python
pr_paths = from_prs[:]